app/controllers/application_controller.rb

```diff
@@ -292,9 +292,15 @@ def access_denied(options ={})
   end
 
   def admin_only_access_denied
-    flash[:error] = ts("Sorry, only an authorized admin can access the page you were trying to reach.")
-    redirect_to root_path
-    false
+    respond_to do |format|
+      format.html do
+        flash[:error] = ts("Sorry, only an authorized admin can access the page you were trying to reach.")
+        redirect_to root_path
+        false
```

Comment:
You can remove `false`. In Rails 6, [renders or redirects will already halt the request cycle](https://guides.rubyonrails.org/v6.0/action_controller_overview.html#filters).